### ISSUE TITLE
[BUG] Instead of spinning, fan dissapears when turned on

### ISSUE THREAD

**Issue by jlafuenti** [ExperienceLovelace/ha-floorplan#408]
**Describe the bug**
When clicking on the fan (which is a switch) the icon of the fan disappears instead of starting to spin

**To Reproduce**
1. Go to dashboard
2. Click on a fan
3. Watch the fan disappear

**Expected behavior**
The fan should spin

**Screenshots**
Can't really take a screenshot of it vanishing

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser firefox and Edge
 - Version FF 121.01, Edge 120.0

**Share configuration**
Please share:
- `YAML` (**IMPORTANT!**)
[floorplan.yaml.txt](https://github.com/ExperienceLovelace/ha-floorplan/files/13995490/floorplan.yaml.txt)

- `CSS`
- 
[floorplan.txt](https://github.com/ExperienceLovelace/ha-floorplan/files/13995494/floorplan.txt)

- SVG-file (**IMPORTANT!**) (Use gofile.io, or something simular)
- 
![floor_1](https://github.com/ExperienceLovelace/ha-floorplan/assets/25994131/9ba2d943-89c4-458c-9797-3dc253719b4d)

_If you create a issue without a SVG example, including the required YAML and CSS to reproduce the issue, it's likely what your issue will be closed, or marked as unresolvable, as it's lacking the proper amount of details. Thank you for understanding the situation._

**Additional context**

**Comment by OzGav** [comment]
This isn’t a bug. Please search before posting https://community.home-assistant.io/t/floorplan-for-home-assistant/17394/1547?page=76

**Comment by jlafuenti** [comment]
Apologies, I did search for a while and couldn't find anything to match my issue.  I had assumed a bug since I more or less directly copied the CSS & configuration of the floorplanner_home example and every other bit worked.  But you were right, it was an issue with the transform being incorrect.  I copied the <switch>_group from the floorplanner example without understanding why it existed in the first place.
https://experiencelovelace.github.io/ha-floorplan/docs/create-svg-file/#animations-not-in-the-right-position This was exactly what I needed.
